notebooks/official/experiments/build_model_experimentation_lineage_with_prebuild_code.ipynb

```diff
@@ -0,0 +1,1229 @@
+{
```

Comment:
<p>I suggest more details about the Artifacts you are creating</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/637/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>

Reply:
<p>I add a more explicit phase</p>